modules/sdk-coin-doge/src/doge.ts

```diff
@@ -78,4 +89,15 @@ export class Doge extends AbstractUtxoCoin {
   ): Promise<CrossChainRecoverySigned<TNumber> | CrossChainRecoveryUnsigned<TNumber>> {
     return super.recoverFromWrongChain(params);
   }
+
+  parseUnspents<TNumber extends number | bigint = bigint>(unspents: Unspent<TNumber>[]): void {
```

Comment:
I suggested a variant of this here that has no side effects https://github.com/BitGo/BitGoJS/pull/2785/files#diff-3bc4bb8653ef7242d182520184d5b1923b3363693cfb6ae0df1c4a6ea7f6263dR85

let's use this pattern instead

```
// types used for talking to WP
type UnspentJSON = { valueString: string; /* ... other fields */ }
type TxInfoJSON = { unspents: UnspentJSON[]; /* other fields */ }

// types used internally
type TxInfo<TNumber> = { unspents: Unspent<TNumber>; /* ... */ };

function decodeTxInfo<TNumber extends number | bigint>(txInfo: TxInfoJSON, amountType: 'number' | 'bigint'): TxInfo<TNumber> {
 return { ...txInfo, unspents: decodeUnspents(txInfo.unspents, amountType) };
}

// similarly we can write `encodeTxInfo` etc
```

In `AbstractUtxoCoin` we should then use `UnspentJSON` / `TxInfoJSON` as the type for WP responses and use `decodeTxInfo` / `encodeTxInfo` in `abstractUtxoCoin.ts` to get the WP data into the correct representation